### ISSUE TITLE
Large space between apostroph alikes and punctuation (Kerning)

### ISSUE THREAD

**Issue by rriemann** [alif-type/libertinus#323]
Dear all,

maybe I do not understand other overarching concerns, but I wonder why the space between apostroph and punction is so large.

Here a comparison for Italian between [Adamina](https://fonts.google.com/specimen/Adamina) and Libertinus Serif from LibreOffice (Linux):

![image](https://user-images.githubusercontent.com/111932/80928872-e8064000-8da7-11ea-83e2-cd75ca707a4a.png)

I'm on Libertinus v6.11-1.1.

**Comment by alerque** [comment]
I would agree the `’.` kerning is too big and could be tightened up.

I just checked the double closing quote kerning and it suffers from the same issue.

![image](https://user-images.githubusercontent.com/173595/80940833-7fd84e00-8de9-11ea-81ed-ea651bce76f0.png)

That being said this is rarely an issue in the English and Turkish texts I work with. Apostrophes rarely end a word as in your example and in English style guides the full stop goes inside the quotes not outside. Still should be fixed, it just might partially explain why it's never been addressed.

**Comment by waldyrious** [comment]
> in English style guides the full stop goes inside the quotes not outside.

That [depends on the style guide](https://en.wikipedia.org/wiki/Quotation_marks_in_English#Order_of_punctuation).

**Comment by alerque** [comment]
Correct, I actually meant to say "most" there but it got lost in other copy editing. It's definitely not 100% normative.

**Comment by rriemann** [comment]
I did some brainstorming to find further pairs with large kerning:

![Screenshot_20200505_001630](https://user-images.githubusercontent.com/111932/81018914-c5d2f780-8e65-11ea-8c3d-1740672f2879.png)

**Comment by khaledhosny** [comment]
I’m not a fan of tighter kerning between closing quote mark an the period. But I agree about the other kerning issues.

**Comment by alerque** [comment]
I think part of the issue there is how inconsistent the kerning is depending on the glyph order. There is a radical difference between `'.` and `.'` and I think the difference is uncalled for. Those should be normalized so that they produce a similar visual density in both orders. Personally I prefer the tighter kerning, but perhaps at least splitting the difference would be in order.

**Comment by Mrifk** [comment]
Could also stand to have tighter kerning on 's

![Screenshot at 2020-06-29 16-08-34](https://user-images.githubusercontent.com/22368066/86051059-d0071180-ba22-11ea-891e-40c00df0f057.png)

**Comment by khaledhosny** [comment]
OK, I’m convinced now. But instead of adjusting kerning, the side bearings of the quotes need to be adjusted.

**Comment by khaledhosny** [comment]
If one wants to do this, 20 inner side bearing and 50 outer side bearing for serif regular seems to be good.
